externs/shaka/player.js

```diff
@@ -190,6 +192,10 @@ shakaExtern.Stats;
  *   cannot be satisfied.
  * @property {!Array.<string>} roles
  *   The roles of the track, e.g. 'main', 'caption', or 'commentary'.
+ * @property {?number} videoId
+ *   (only for variant tracks) The video track id.
```

Comment:
Nitpick: please change to video STREAM id (instead of track).
A track is a representation of a variant or text stream, so technically we don't have video/audio tracks.
Same for audio below. The rest of the CL looks good to me.